tsl/test/expected/compression_bgw.out

```diff
@@ -205,3 +205,66 @@ ERROR:  permission denied to start compress_chunks background process as role "n
 \set ON_ERROR_STOP 1
 RESET ROLE;
 REVOKE NOLOGIN_ROLE FROM :ROLE_DEFAULT_PERM_USER;
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DROP TABLE IF EXISTS conditions CASCADE;
+NOTICE:  drop cascades to table _timescaledb_internal.compress_hyper_2_4_chunk
+CREATE TABLE conditions(
+    time TIMESTAMPTZ NOT NULL,
+    device INTEGER,
+    temperature FLOAT
+);
+SELECT * FROM create_hypertable('conditions', 'time', 'device', 30,
+       chunk_time_interval => '1 day'::interval,
+       migrate_data => true);
```

Comment:
Why is `migrate_data` set? The table is empty.

Reply:
Copy-paste.

Reply:
Removed.